packages/python/plotly/_plotly_utils/basevalidators.py

```diff
@@ -2641,34 +2636,31 @@ def validate_coerce(self, v, skip_invalid=False, _validate=True):
             for v_el in v:
 
                 if isinstance(v_el, BaseTraceType):
-                    # Clone input traces
-                    v_el = v_el.to_plotly_json()
+                    v_el = v_el._props
```

Comment:
```suggestion
                    if isinstance(v_el, Histogram2dcontour):
                        v_el = dict(type="histogram2dcontour", **v_el._props)
                    else:
                        v_el = v_el._props
                    
```
Seems like the check you called out [below](https://github.com/plotly/plotly.py/pull/3946/files#r1012659585) got broken at some point in the past, but it would work if we put it up here, right? And since this is a deprecation fallback I'm not worried about performance.